### PR TITLE
[plugin.video.fosdem] 0.0.6

### PR DESCRIPTION
**Pull request by jelly**
### Description

Kodi addon for Fosdem videos.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.fosdem/addon.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Videos of FOSDEM talks</summary>
+        <description lang="en_GB">This addon provides access to videos published on https://video.fosdem.org</description>
+        <language>en</language>
+        <platform>all</platform>
+        <reuselanguageinvoker>true</reuselanguageinvoker>
+        <license>MIT</license>
+        <forum></forum>
```
Comment (enen92) [inline plugin.video.fosdem/addon.xml]: Can you drop this if empty?

File: plugin.video.fosdem/addon.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <website>https://fosdem.org</website>
+        <email>jelle@vdwaa.nl</email>
+        <source>https://github.com/jelly/plugin.video.fosdem</source>
+	<news>v0.0.1 (2018-5-17)
+- Initial release
```
Comment (enen92) [inline plugin.video.fosdem/addon.xml:21]: You miss new version in the news section. Btw changelog.txt will again be used as fallback if news is empty in nexus

File: plugin.video.fosdem/resources/lib/addon.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, unicode_literals
```
Comment (enen92) [inline plugin.video.fosdem/resources/lib/addon.py:2]: Dummy init empty files are no longer needed in python3

File: plugin.video.fosdem/resources/lib/addon.py [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    # Determine if FOSDEM happened this year already
+    if now.month < 2 and now.day < 3:
+        year -= 1
+
+    # Range does not include the end.
+    year += 1
+    return range(year - YEARS_SHOWN, year)
+
+
+def get_setting_int(name):
```
Comment (enen92) [inline plugin.video.fosdem/resources/lib/addon.py]: Not needed we have xbmcaddon.getSettingInt https://codedocs.xyz/xbmc/xbmc/group__python__xbmcaddon.html#ga95d7941b2ec77a6fc2427aa7a98f4214

File: plugin.video.fosdem/resources/settings.xml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (enen92) [inline plugin.video.fosdem/resources/settings.xml]: Would be nice if you can convert to the new settings format https://kodi.wiki/view/Add-on_settings_conversion
There's an add-on to do it automatically: https://forum.kodi.tv/showthread.php?tid=350135

Comment (jelly) [reply plugin.video.fosdem/resources/settings.xml]: The automatic conversion failed for me, so I've done it manually.

File: plugin.video.fosdem/resources/lib/addon.py (hunk 4 above)
Comment (jelly) [reply plugin.video.fosdem/resources/lib/addon.py]: Thanks, fixed!

File: plugin.video.fosdem/addon.xml (hunk 1 above)
Comment (jelly) [reply plugin.video.fosdem/addon.xml]: Done.

File: plugin.video.fosdem/resources/lib/addon.py (hunk 3 above)
Comment (jelly) [reply plugin.video.fosdem/resources/lib/addon.py:2]: Since there is still python2 compat in general, can we keep it for now?

Comment (enen92) [reply plugin.video.fosdem/resources/lib/addon.py:2]: That's fine

**Review by enen92 — APPROVED** [review]

**Pull request merged**